addons/resource/models/resource_resource.py

```diff
@@ -144,8 +144,9 @@ def _get_unavailable_intervals(self, start, end):
             calendar_mapping[resource.calendar_id or resource.company_id.resource_calendar_id] |= resource
 
         for calendar, resources in calendar_mapping.items():
-            resources_unavailable_intervals = calendar._unavailable_intervals_batch(start_datetime, end_datetime, resources, tz=timezone(calendar.tz))
-            resource_mapping.update(resources_unavailable_intervals)
+            if calendar:
```

Comment:
you can do an early continue to avoid the need to indent the next lines:
```suggestion
            if not calendar:
                continue
```

Reply:
Hello @Bertrand2 :wave: 
Changes are done.
Can you have a look?
Thanks!